lib/nifty_services/support/string.rb

```diff
@@ -0,0 +1,14 @@
+unless String.method_defined?(:underscore)
+  class String
+   def underscore
```

Comment:
@lucasgomide I think we can improve this method, for example, why are you using `freeze` in plain string objects? (I mean, this is usefull when you want to freeze a object who is assigned to a variable)
After this, ActiveSupport dependency will be removed \o/, thanks!